spack/package.py

```diff
@@ -12,19 +11,19 @@ class Arbor(CMakePackage, CudaPackage):
 
     homepage = "https://arbor-sim.org"
     git = "https://github.com/arbor-sim/arbor.git"
-    url = "https://github.com/arbor-sim/arbor/releases/download/v0.8.1/arbor-v0.9.0-full.tar.gz"
+    url = "https://github.com/arbor-sim/arbor/releases/download/v0.10.1/arbor-v0.10.0-full.tar.gz"
```

Comment:
The PATCH part of the semver are not synced up.

Reply:
Yes, that's due to a misunderstanding between me and GitHub's release mechanisms.